docs/cloud/core-concepts.md

```diff
@@ -82,58 +80,54 @@ Examples of fields:
    </td>
-  </tr>
-  <tr>
-   <td>Destination default
+   </tr>
+   <tr>
+   <td>Mirror source structure
    </td>
-   <td>All streams will be replicated and stored in the default namespace defined on the destination settings page. For settings for popular destinations, see<a href="https://docs.airbyte.com/understanding-airbyte/namespaces#destination-connector-settings"> ​​Destination Connector Settings</a>
+   <td>Some sources (for example, databases) provide namespace information for a stream. If a source provides namespace information, the destination will mirror the same namespace when this configuration is set. For sources or streams where the source namespace is not known, the behavior will default to the "Destination default" option. 
    </td>
-  </tr>
-  <tr>
+   </tr>
+   <tr>
    <td>Custom format
    </td>
-   <td>All streams will be replicated and stored in a user-defined custom format. See<a href="https://docs.airbyte.com/understanding-airbyte/namespaces#custom-format"> Custom format</a> for more details.
+   <td>All streams will be replicated to a single user-defined namespace. See<a href="https://docs.airbyte.com/understanding-airbyte/namespaces#--custom-format"> Custom format</a> for more details
    </td>
-  </tr>
-</table>
+   </tr>
+   </table>
 
 ## Connection sync modes
 
 A sync mode governs how Airbyte reads from a source and writes to a destination. Airbyte provides different sync modes to account for various use cases.
 
-- **Full Refresh | Overwrite:** Sync all records from the source and replace data in destination by overwriting it.
-- **Full Refresh | Append:** Sync all records from the source and add them to the destination without deleting any data.
-- **Incremental Sync | Append:** Sync new records from the source and add them to the destination without deleting any data.
-- **Incremental Sync | Append + Deduped:** Sync new records from the source and add them to the destination. Also provides a de-duplicated view mirroring the state of the stream in the source.
+- **Full Refresh | Overwrite:** Sync all records from the source and replace data in destination by overwriting it each time.
+- **Full Refresh | Append:** Sync all records from the source and add them to the destination without deleting any data. This creates a historical copy of all records each sync.
+- **Incremental Sync | Append:** Sync new records from the source and add them to the destination without deleting any data. This enables efficient historical tracking over time of data. 
+- **Incremental Sync | Append + Deduped:** Sync new records from the source and add them to the destination. Also provides a de-duplicated view mirroring the state of the stream in the source. This is the most common replication use case. 
 
 ## Normalization
```

Comment:
I think this section should be re-written to be about typing and deduplication, then we can say that all of these destination are done or coming soon

Reply:
We can do this in another PR too

Reply:
Yep, makes sense! Will handle in another PR.